app/views/how-tos/github/store-new-prototype.html

```diff
@@ -0,0 +1,81 @@
+{% extends "layout.html" %} {% block pageTitle %} Store your new prototype in
+GitHub - NHS prototype kit {% endblock %} {% block beforeContent %} {{
+breadcrumb({ items: [ { href: "/", text: "Home" }, { href: "/how-tos", text:
+"Guides" }, { href: "/how-tos/github/index", text: "Store code online with
+GitHub and GitHub desktop" } ] }) }} {% endblock %} {% block content %}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-three-quarters">
+    <h1 class="nhsuk-heading-xl">Store your new prototype in GitHub</h1>
+
+    <p>
+      To store your new prototype in GitHub, you need to
+      <a href="index">set up GitHub Desktop</a>. If you work for NHS England,
```

Comment:
Probably don't want to link to index

Reply:
I was being lazy and using relative urls - have changed to absolute (it's the index page for this section, not the home page)